### ISSUE TITLE
Liquid Magma Totem - spell cooldown - not able to show cooldown

### ISSUE THREAD

**Issue by thach1ef** [WeakAuras/WeakAuras2#1120]
**Describe the bug**
When creating a WA for Liquid Magma Totem using spell cooldown progress, show always, you are unable to utilize the cooldown section on page 1. It is disabled.

**Do you have an error log of what happened?**
No errors

**To Reproduce**
Create WA with the following trigger. Status > CD progress spell > Name: Liquid Magma Totem > Show Always

Go to page 1. Notice the cooldown section is greyed out and you cannot use.

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
Yes

**Which version of WeakAuras are you using?**
2.11.0

**Comment by asakawa-k** [comment]
I've tried a few different combinations of being on Shaman, in various specs, or other classes, and can't reproduce.     
Did you actually go through these steps, after encountering your issue, to see if it was reproducible? Or did you just write down what you remembered doing when it occurred?

**Comment by thach1ef** [comment]
I have that exact WA setup right now working that way. I did not start from
scratch, but I can do that and report back.

**Comment by asakawa-k** [comment]
Yeah, give it a go. And make sure you've really disabled all other Addons. 
Also, you could paste the import string of the problematic Aura you have, here.

**Comment by thach1ef** [comment]
Will try from scratch, here is the problem WA:

!DAv3YjUnu4NMot7f7oXK0nj9oG1zd7aMuKZU9kbcBzB1il5Ajtc5cE275ijdo)St3PmmGS85)Z35tIgrtPebn56lJo7YRPedLm6JrrF8mkPMsYXVGaw(t2r3PfklLmnojnELFVOB0klr8mNMenIsEAzrHHBPjF4CWuzDgRUofeJso81dFLs2F89rN)jkHPYQ0TVYSzATmx)OI8OOHtNm0m33KZSCkPl8FVOX5LC6wWEzwHwzWa2yzTwCrUEDPu)i(A8)1fTSAq1VZzpmURLz(Jyj)WgsfRMPoS5dh2mx8pDI8dBwWkRzh2KQT86dBwXz57dMW7gWhvGDHiVqOeMkWzWAyP1Tcw3z4O8t1sDl6Fin5TkM8B8wdAGKOOqvmwX2k55y2(aN3m20WZSRyGBq9mCzXBksOntbt4(oY975UFVa)1gm8uO9WeQAoQ8SeYSphtjnT6YwUXCxlpteIKG8b3mzzA6YfRM9LB779yF(gjReGhlVpD(SeWovCrzf0mXwzUW0iz7NLHw73JIUCeSPuZCOhHQSv31GlR7KwrFbktYm(UfiHe6kKBhVyCYjPilg)LyOQ0xoxBzsmvM4FYRp8qllJ)gRJsGfsuadcrFTaEJn6hS)G4YHT7fj5QHb0oELit6qFoFnuPb6eb6aDPjytkODUOOqKbcS)nbqZUMtXWBYPGlVxGof3OHB3YSwFuueqNVwZwT8nvaeVInSjEGTZEcyK)QIBVCU2o(V(2vH2)7IJqDs3ddPKTDwRwTChVfWadygOKBAfpFyZF2XYBzwCCkfh7f1S(G84GQ1zjLwXrekhPjGKjhEkhMtrXx7fXWHyo3GrFnetVw3FS0hNt))4lBpD1nibIpTjtxfhNCSe9Fos2Ylbt711v4jmztfZbvyDwW4lB6zWSdg88eSJUyqL91ZIHP9Fowbk5xAgy(3Vl9SwxttoliNxfii19OLFcNHyPFg61Gp6zbrumwKjwmACvNhf52Qatd3WSyxYb3HQyHOSN7faGg322wrzj8Gp(spUbUehz6ffZfUI3kYiaz(YaP(y5JS9q1vbbrWeK3nUbNch)Suj3F6GQ1vceizf1qQGKcszcEKts01JGpbEmuDhqylo9uum09lvlegKfbtw8LEeZTXZV7M7N3JEHbhBheKD4joeK8fsp4j(o3qQRv55a9Mgzb726ElbSP4jkz90XK01K0XRGYsqVPHZupS5UWXeh28Reml(nWZyy7YMxwpuhBKTCMKCmNF)QwpZCqOjHQ0S8aUVFgKsI8Y6sXbXpCWLl(j3fpF(GcAV5WnEzBDIJ8unafyErbV)49FmwyyJm6tNff1ZIB(7ofq5UJJee79xczhp17Pf6CEyyajse9t3Eqzwfp7HHOteOTJ1kW5aaAP6Bg4UYooooIwQIPk5NmJ)DtChS3WBTH7PeO4TiurK9W(p3xw3E6Mt4TQoD5dq0DGnVyegW0)9d

**Comment by asakawa-k** [comment]
@InfusOnWoW /devs, looks like this trigger had previously been set to be a Totem trigger, triggering on ***Inverse***. 
Even after the user changed the trigger to the CD progress trigger, that "inverse" setting was having an impact. 

The quick way to fix it @thach1ef , is to change the trigger to Status - Totem and untick Inverse, then change it back to CD Progress.
